src/test/java/subway/StationAcceptanceTest.java

```diff
@@ -3,65 +3,152 @@
 import io.restassured.RestAssured;
 import io.restassured.response.ExtractableResponse;
 import io.restassured.response.Response;
+import org.assertj.core.api.Assertions;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
 import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.MediaType;
 
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
+import java.util.stream.Stream;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
 @DisplayName("지하철역 관련 기능")
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
 public class StationAcceptanceTest {
+
+    private final String STATION_API_URL = "/stations";
+
+    static Stream stationNames() {
+        return Stream.of("강남역");
+    }
+
     /**
      * When 지하철역을 생성하면
      * Then 지하철역이 생성된다
      * Then 지하철역 목록 조회 시 생성한 역을 찾을 수 있다
      */
     @DisplayName("지하철역을 생성한다.")
-    @Test
-    void createStation() {
+    @ParameterizedTest
+    @MethodSource("stationNames")
+    void createStation(String stationName) {
         // when
-        Map<String, String> params = new HashMap<>();
-        params.put("name", "강남역");
+        Map<String, String> parameter = new HashMap<>();
+        parameter.put("name", stationName);
 
         ExtractableResponse<Response> response =
-                RestAssured.given().log().all()
-                        .body(params)
-                        .contentType(MediaType.APPLICATION_JSON_VALUE)
-                        .when().post("/stations")
-                        .then().log().all()
+                RestAssured
+                        .given()
+                            .log()
+                                .all()
```

Comment:
```suggestion
                        .given().log().all()
```

이렇게 하는게 오히려 좋은 것 같아요 😅 
개념적으로 분리해서 보아야하는 부분이 아니라고 생각합니다.
메소드의 행이 늘어나서 오히려 가독성이 떨어지는 것 같아요 😢